lib/jekyll-import/importers/wordpressdotcom.rb

```diff
@@ -30,7 +30,7 @@ def self.download_images(title, post_hpricot, assets_folder)
         images.each do |i|
           uri = i["src"]
 
-          i["src"] = assets_folder + "/" + File.basename(uri)
+          i["src"] = "{{ site.baseurl }}/%s/%s" % [assets_folder, File.basename(uri)]
```

Comment:
Yes, that would make more sense and it's a platform indenpendent way. I'll fix the PR tomorrow evening (EEST).